provider/k8s/template/app/service.yml.tmpl

```diff
@@ -8,10 +8,23 @@ metadata:
     service: {{.Service.Name}}
 type: Opaque
 data:
-  {{ range env .Environment }}
+  {{ range keyValue .Environment }}
   {{.Key}}: {{ safe (base64 .Value) }}
   {{ end }}
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    {{ range keyValue .Annotations }}
+    {{.Key}}: "{{ quoteEscape .Value}}"
+    {{ end }}
+  namespace: {{.Namespace}}
+  name: serviceaccount-{{.Service.Name}}
```

Comment:
Seems like the name could just be the service name since it will already be namespaced by "serviceaccount", e.g.

`kubectl get serviceaccount/web2`